app/consumer/ante_handler.go

```diff
@@ -42,7 +42,7 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 	anteDecorators := []sdk.AnteDecorator{
 		ante.NewSetUpContextDecorator(),
 		ante.NewExtensionOptionsDecorator(nil),
-		consumerante.NewMsgFilterDecorator(options.ConsumerKeeper),
+		consumerante.NewMsgFilterDecorator(options.ConsumerKeeper, "/cosmos.evidence", "/cosmos.slashing"),
```

Comment:
Adding the filtering logic in the `MsgFilterDecorator` would require all the consumer chains to update their application, i.e., adding this line to their `ante_handler.go`. An alternative is to add the logic directly to the `DisabledModulesDecorator`, which would require consumer chains to only upgrade their version of ICS.

Reply:
hmmm make sense, i will move the logic to `DisabledModulesDecorator`

Reply:
updated as request